confd/templates/opennms.properties.d/core.ipc.properties.tpl

```diff
@@ -0,0 +1,83 @@
+#
+# DO NOT CHANGE THIS FILE! GENERATED FROM confd
+#
+#
+{{if exists "/core/ipc/sink/strategy"}}
+#
+# Global Sink Settings
+org.opennms.core.ipc.sink.strategy={{getv "/core/ipc/sink/strategy"}}
+org.opennms.core.ipc.sink.initialSleepTime={{getv "/core/ipc/sink/initialsleeptime" "60000"}}
+{{end}}
+{{if exists "/core/ipc/rpc/strategy"}}
+#
+### Global RPC Settings
+org.opennms.core.ipc.rpc.strategy={{getv "/core/ipc/rpc/strategy"}}
+{{end}}
+{{if or (and (exists "/core/ipc/sink/strategy") (eq (getv "/core/ipc/sink/strategy" "") ("sqs"))) (and (exists "/core/ipc/rpc/strategy") (eq (getv "/core/ipc/rpc/strategy" "") ("sqs")))}}
+#
+# Global AWS Settings for RPC and Sink
+org.opennms.core.ipc.aws.sqs.aws_region={{getv "/core/ipc/aws/sqs/aws/region"}}
+org.opennms.core.ipc.aws.sqs.aws_access_key_id={{getv "/core/ipc/aws/sqs/aws/access/key/id"}}
+org.opennms.core.ipc.aws.sqs.aws_secret_access_key={{getv "/core/ipc/aws/sqs/aws/secret/access/key"}}
+{{end}}
+{{if and (exists "/core/ipc/rpc/strategy") (eq (getv "/core/ipc/rpc/strategy" "") ("kafka"))}}
+#
+# Kafka RPC Settings
+org.opennms.core.ipc.rpc.kafka.bootstrap.servers={{getv "/core/ipc/rpc/kafka/bootstrap/servers" "127.0.0.1.9092"}}
```

Comment:
`bootstrap.servers` is actually a Kafka Producer/Consumer property according to their documentation:
https://kafka.apache.org/documentation/

Is it possible to be able to "auto-magically" map *all* the properties defined on a "parent" element to be included ? BTW, this is how Kafka and Flows can be configured for the docker images on Minion and Sentinel.

That way, you can include which every Kafka property you need to either Sink or RPC or both without the need to modify the templates every time you need a new variable.

For example, [here](https://github.com/OpenNMS/opennms-drift-kubernetes/blob/master/config/onms-core-init.sh#L151) is how I setup Kafka for OpenNMS. It would be super useful if we can handle any property instead of explicitly defining each of them on the `confd` templates.